### DISCUSSION TITLE
Setting the active tab of a Tabs object

### DISCUSSION THREAD

**Discussion by Rexxt** [Textualize/textual#2147] [General]
Hello community, I'd like to know how to set the active tab of a Tabs object to be its last tab.

On a second note, when I click on a tab, I'd also like to remove all tabs after it.

Does anyone have an idea on how to do that? Thanks in advance.

**Comment by TomJGooding** [comment]
>  I'd like to know how to set the active tab of a Tabs object to be its last tab.

From the docs it looks like there's an `active` argument for the "ID of the tab which should be active on start", if that's what you mean?

> On a second note, when I click on a tab, I'd also like to remove all tabs after it.

You could use the `remove_tab` method or `clear` and render the widget again maybe?

**Comment by Rexxt** [comment]
yeah that might be it, i'll try later, thanks for the help 👍
